scopes/react/react/react.env.ts

```diff
@@ -67,7 +67,7 @@ const prettierConfig = require('./prettier/prettier.config.js');
 type CompilerMode = 'build' | 'dev';
 
 type GetBuildPipeModifiers = PipeServiceModifiersMap & {
-  tsModifier?: PipeServiceModifier;
+  compilerModifier?: PipeServiceModifier;
```

Comment:
This is a breaking change, you can't just change it. it will break many users
also, I really want to remove this soon, getBuildPipe shouldn't get transformers (like the other get*** that shouldn't get them). 
let's undo this change for now, and discuss everything around build pipeline seperately

Reply:
I dont think it is a breaking change, it's local to the env and only used here

Reply:
And i agree, I have no idea why it gets transformers and where it gets them from...

Reply:
(when i say it's only used here, it's only used by the UseTypescript function in main.runtime, which has also been updated)